### ISSUE TITLE
No convergence in train.py

### ISSUE THREAD

**Issue by Mahasweta-usc** [zihangdai/xlnet#234]
My parameters:
Used the same parameters suggested for xlnet pretrain. Gradually reduced learning rate when loss kept swinging between ~1.2 and ~0.2 after 40,000 epochs
